recipe/meta.yaml

```
@@ -1,108 +1,113 @@
 {% set name = "scikit-learn" %}
-{% set version = "1.6.1" %}
+{% set version = "1.7.0" %}
```

```suggestion
{% set version = "1.7.1" %}
```
A new version was just released